docs/snippets/vue/msw-addon-configure-handlers-graphql.ts.mdx

```diff
@@ -41,43 +45,40 @@ const TestData = {
 export default meta;
 type Story = StoryObj<typeof DocumentScreen>;
 
-/*
- *👇 Render functions are a framework specific feature to allow you control on how the component renders.
- * See https://storybook.js.org/docs/api/csf
- * to learn how to use render functions.
- */
 export const MockedSuccess: Story = {
-  render: () => ({
-    components: { DocumentScreen, WrapperComponent },
-    template: '<WrapperComponent><DocumentScreen /></WrapperComponent>',
-  }),
   parameters: {
-    msw: [
-      graphql.query('AllInfoQuery', (req, res, ctx) => {
-        return res(ctx.data(TestData));
-      }),
-    ],
+    msw: {
+      handlers: [
+        graphql.query('AllInfoQuery', () => {
+          return new HttpResponse.json({
+            data: {
+              allFilms: {
+                films,
+              },
+            }
+          });
+        }),
+      ],
+    },
   },
 };
 
 export const MockedError: Story = {
-  render: () => ({
-    components: { DocumentScreen, WrapperComponent },
-    template: '<WrapperComponent><DocumentScreen /></WrapperComponent>',
-  }),
   parameters: {
-    msw: [
-      graphql.query('AllInfoQuery', (req, res, ctx) => {
-        return res(
-          ctx.delay(800),
-          ctx.errors([
-            {
-              message: 'Access denied',
-            },
-          ]),
-        );
-      }),
-    ],
+    msw: {
+      handlers: [
+        graphql.query('AllInfoQuery', async () => {
+          delay(800);
```

Comment:
This should be awaited
Also don't forget to add `HttpResponse` and `delay` to the msw imports at line 4